packages/core/src/renderer/createDOMRenderer.ts

```diff
@@ -67,14 +67,17 @@ export function createDOMRenderer(
             metadata,
           );
 
+          if (process.env.NODE_ENV !== 'production' && isDevToolsEnabled) {
+            if (!debugData.hasCSSRule(ruleCSS)) {
+              debugData.addCSSRule(ruleCSS);
+            }
+          }
+
```

Comment:
As it's SSR specific I was wondering if we could fill data in `rehydrateRendererCache()` instead:

https://github.com/microsoft/griffel/blob/79c97b9ec6b4b028e31e17b613e6d89a37bfd6a7/packages/core/src/renderer/rehydrateRendererCache.ts#L41-L45

Will it work?